src/test/java/org/folio/rest/impl/MailConfigTest.java

```diff
@@ -49,6 +52,9 @@ public void messageShouldIncludeAuthMethodsFromConfiguration() throws Exception
       "2500", AUTH_METHODS);
     initModConfigStub(mockServer.port(), configurations);
 
+    SmtpConfiguration smtpConfiguration = createSmtpConfiguration("user", "pws", "localhost", 2500,
+      AUTH_METHODS);
+
```

Comment:
Create tests:
1. Db conf exists and valid
  a. remote conf exists and valid
  b. remote conf exists and not valid
  c. remote conf doesn't exist
2. Db conf exists and not valid
  a, b, c
3. Db conf doesn't exist
  a, b, c